ghcup-cross-0.0.8.yaml

```diff
@@ -138,3 +138,16 @@ ghcupDownloads:
               dlOutput: ghc-9.10.0.20240412-x86_64-linux-alpine3_18-wasm-cross_wasm32-wasi-release+fully_static.tar.xz
           Linux_Alpine:
             unknown_versioning: *ghc-wasm32-wasi-9101-a3-64-static
+    wasm32-wasi-9.10.1:
+      viTags:
+      - base-4.20.0.0
+      viArch:
+        A_64:
+          Linux_UnknownLinux:
+            unknown_versioning: &ghc-wasm32-wasi-9101-64-static
+              dlHash: 791f50ad71f28a5febd10e5b7384545c74d87805228a5e541d801aaab727448a
+              dlSubdir: ghc-9.10.1-wasm32-wasi
+              dlUri: https://github.com/tweag/ghc-wasm-bindists/releases/download/20240609T200531/wasm32-wasi-ghc-9.10.tar.xz
+              dlOutput: ghc-x86_64-linux-alpine3_18-wasm-cross_wasm32-wasi-release+fully_static.tar.xz
```

Comment:
This is not a unique tarball name. It must include the proper version, otherwise we're going to hit download/checksum errors.